deploy/cd-namespace/chart/values.kind.yaml

```diff
@@ -0,0 +1,26 @@
+# General
+serviceAccountName: 'default'
+
```

Comment:
`API_HOST` should be set from "openshift.default.svc.cluster.local" to "kubernetes.default.svc.cluster.local"

Reply:
Right, forgot that one, will add.